src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -100,27 +99,38 @@ private static string BuildProcDumpArgs(int processId, string filename)
             // -accepteula: Auto accept end-user license agreement
             // -t: Write a dump when the process terminates.
             // This will create a minidump of the process with specified filename
-            return "-accepteula -t " + processId + " " + filename + ".dmp";
+            return "-accepteula -e 1 -g -t -f STACK_OVERFLOW -f ACCESS_VIOLATION " + processId + " " + filename + ".dmp";
```

Comment:
Are we okay with -g option @abhishkk

Reply:
@kaadhina :
@vagisha-nidhi did extensive validations with known crash scenarios in different possible scenarios. Also she validated if the dump created is usable.